plugin/grails-app/taglib/grails/plugin/springsecurity/SecurityTagLib.groovy

```diff
@@ -201,11 +201,18 @@ class SecurityTagLib implements GrailsConfigurationAware {
 	 * Renders the link if the user has access to the specified URL.
 	 */
 	def link = { attrs, body ->
+		boolean isFallback = isFallback(attrs)
 		// retain original attributes for later, since hasAccess() removes ones necessary to create a link
 		def origAttrsMinusExpression = [:] + attrs
 		origAttrsMinusExpression.remove 'expression'
-		if (hasAccess(attrs, 'link')) {
+		boolean hasAccess = hasAccess(attrs, 'link')
```

Comment:
I think it's better to leave `hasAccess` inline since this is the only place it is used.